foia_hub/templates/about.html

```diff
@@ -20,6 +20,7 @@
             </p>
 
             <h2>Features</h2>
+            <p></p>
```

Comment:
It fixes the line spacing problem via https://github.com/18F/foia-hub/issues/436 I guess we could also use a line break.